services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -289,7 +300,9 @@ export const SubmissionSummary = (): React.ReactElement => {
                                 current status.
                             </Grid>
                         ) : (
-                            <MultiColumnGrid columns={3}>
+                            <MultiColumnGrid
+                                columns={showUndoWithdrawBtn ? 2 : 3}
```

Comment:
Looking at [figma](https://www.figma.com/design/frKNnm6lkcpdjJ2eVJE8Bj/Managed-Care-Review?node-id=18005-79095&t=ncfFVOhwrmz5f9zC-0), this should still be 3 columns.

```suggestion
                                columns={3}
```